src/Bicep.LangServer/Handlers/BicepDocumentSymbolHandler.cs

```diff
@@ -81,6 +81,9 @@ private SymbolKind SelectSymbolKind(DeclaredSymbol symbol)
                 case ResourceSymbol resource:
                     return SymbolKind.Object;
 
+                case ModuleSymbol module:
+                    return SymbolKind.Object;
```

Comment:
There exists `SymbolKind.Module` btw. The icon actually happens to be identical, but it's probably helpful for accessibility.

Icons are at https://code.visualstudio.com/api/references/icons-in-labels (the ones that start with "symbol-")

Reply:
🤦 how did I miss that! I even looked through the enum list...